cd/utils/mxnet_base_image.sh

```diff
@@ -31,7 +31,7 @@ case ${mxnet_variant} in
     echo "nvidia/cuda:11.0-cudnn8-runtime-ubuntu16.04"
     ;;
     cu112*)
-    echo "nvidia/cuda:11.2-cudnn8-runtime-ubuntu16.04"
+    echo "nvidia/cuda:11.2.1-cudnn8-runtime-ubuntu16.04"
```

Comment:
ubuntu 16.04 is EOL by April, 2021 (https://ubuntu.com/about/release-cycle), Can we please move to ubuntu 18.04 or 20.04?

Reply:
Moved to ubuntu 18.04 for all docker base images